src/server/routes/pollRouter.test.ts

```diff
@@ -0,0 +1,122 @@
+import request = require("supertest");
+import app from "../app";
+import db, { Poll, PollInput } from "../models/database";
+
+describe("Test GET /api/polls", () => {
+  // adds test data before each test
+  beforeEach(() => {
+    db.insertPoll({
+      creatorName: "Jed",
+      description: "hey there",
+      options: ["bean bags"],
+      pollName: "test"
+    });
+    db.insertPoll({
+      creatorName: "James",
+      description: "testing again",
+      options: ["banana", "orange"],
+      pollName: "fruit"
+    });
+  });
+  afterEach(() => {
+    // cleans up test data after each test
+    db.removeAllPollsData();
+    db.resetCount();
+  });
+  test("GET should respond with 200 and json in body", async () => {
+    const response = await request(app)
+      .get("/api/polls")
+      .expect("Content-Type", /json/);
+    // const responseCleaned = response.body.map((poll: any) => {
+    //   return { name: poll.name, creator: poll.creator };
+    // });
+    expect(response.status).toBe(200);
+  });
+  test("GET should respond with array of polls", async () => {
+    const response = await request(app).get("/api/polls");
+    expect(response.body).toMatchObject([
+      {
+        creatorName: "Jed",
+        description: "hey there",
+        options: [{ optionId: "1", value: "bean bags", votes: [] }],
+        pollId: "1",
```

Comment:
hard-coding id values would not work in normal UUID generated implementation, so better to restructure this to handle id checking as `notNull` condition.  There are test tricks you can use, like 'check objects contain this subset of data' which doesn't contain id, and then iterate over the array checking all `id`s are non-null.